src/module/migration/migrations/818-spin-tale-adventure-specific.ts

```diff
@@ -13,20 +13,10 @@ export class Migration818SpinTaleAdventureSpecific extends MigrationBase {
             const oldSpinTaleId = "Compendium.pf2e.adventure-specific-actions.5gahZQXf3UVwATSC";
             const newSpinTaleId = "Compendium.pf2e.actionspf2e.hPZQ5vA9QHEPtjFW";
 
-            if (source.system.description.value) {
+            if (source.system.description.value && source.system.description.value.length > 0) {
```

Comment:
These are equivalent checks. I don't understand what you're trying to do here.

Reply:
I guess I misunderstood your initial comment.

Was just checking for undefined description.value, since Migration799RMRecallKnowledgeDuplicates seemed to indicate that was possible. Although I guess since it runs first it will fix it.